src/components/vms/hostvmslist.jsx

```diff
@@ -93,6 +105,29 @@ const HostVmsList = ({ vms, config, ui, storagePools, actions, networks, onAddEr
                     .map(state => { return { value: rephraseUI('resourceStates', state), apiState: state } })
                     .sort((a, b) => (prioritySorting[a.apiState] || 0) - (prioritySorting[b.apiState] || 0) || a.value.localeCompare(b.value)));
 
+    useState(() => {
```

Comment:
useState is not supposed to be used like this, in fact what you need here a useEffect hook and options as a dependency.

You can replace the whole useState body with this:
```
    useEffect(() => {
        setStatusSelected(getStatusFilterOption(options));
    }, [options]);
```